istio-telemetry/prometheus/templates/inrgess.yaml

```diff
@@ -1,5 +1,5 @@
 {{- if .Values.prometheus.ingress.enabled -}}
```

Comment:
Can you help to correct this file name from `inrgess`-->`ingress` along with your changes? Thanks

Reply:
Oh yes didn't noticed this one.